src/neuroglancer/segmentation_user_layer.ts

```diff
@@ -289,6 +303,9 @@ class SegmentationUserLayerDisplayState implements SegmentationDisplayState {
   selectSegment = this.layer.selectSegment;
   transparentPickEnabled = this.layer.pick;
   baseSegmentColoring = new TrackableBoolean(false, false);
+  baseSegmentHighlighting = new TrackableBoolean(false, false);
+  useTempSegmentStatedColors2d =
+      this.layer.registerDisposer(SharedWatchableValue.make(this.layer.manager.rpc, false)); // where does this line belong? Also should it be in segmentationColorGroupState?
```

Comment:
I think it depends on whether the temporary colors should also apply to other layers for which the segment color state is linked to this layer.  Probably not too likely to matter either way, so I think it is fine to leave it here for now.